### DISCUSSION TITLE
Is the SYNC plugin mandatory? I get constant errors around missing SYNC_* settings

### DISCUSSION THREAD

**Discussion by meeowgler** [netalertx/NetAlertX#832] [General]
Is the SYNC plugin mandatory? I currently do not have it enabled.

I get constant errors around missing SYNC_* settings. Example:
ERROR - setting_missing - Setting not found for key: SYNC_node_name in file /app/front/api/table_settings.json

Thanks!

**Comment by jokob-sk** [comment]
Hi @meeowgler ,

Thanks for the question.

Yes, it shouldn't be possible to disable the SYNC plugin. I included the option to mark is at disabled by mistake. This should be fixed in the next release.
